### ISSUE TITLE
Check if you are using openrestry-build-tools for having all the functionality

### ISSUE THREAD

**Issue by alesanmed** [bitnami/bitnami-docker-kong#3]
**Description**

When I enable the TCP stream listen I get an error in the error log and the TCP stream doesn't work.

**Steps to reproduce the issue:**

1. Enable stream listen via ENV variables
2. Launch kong
3. Go to the error logs
4. See the error

**Describe the results you received:**

```
2020/11/17 07:26:12 [error] 98#0: *56007 lua entry thread aborted: runtime error: /opt/bitnami/kong/openresty/lualib/resty/kong/tls.lua:299: /opt/bitnami/kong/openresty/luajit/lib/libluajit-5.1.so.2: undefined symbol: ngx_stream_lua_kong_ffi_proxy_ssl_disable
stack traceback:
coroutine 0:
	[C]: in function '__index'
	/opt/bitnami/kong/openresty/lualib/resty/kong/tls.lua:299: in function 'disable_tls'
	.../openresty/luajit/share/lua/5.1/kong/runloop/handler.lua:883: in function 'balancer_prepare'
	.../openresty/luajit/share/lua/5.1/kong/runloop/handler.lua:1057: in function 'before'
	...itnami/kong/openresty/luajit/share/lua/5.1/kong/init.lua:621: in function 'preread'
	preread_by_lua(nginx-kong-stream.conf:79):2: in main chunk while prereading client data, client: 100.102.125.64, server: 0.0.0.0:9759
```

**Describe the results you expected:**

The stream listen should work without problems

**Additional information you deem important (e.g. issue happens only occasionally):**

I opened [an issue](https://github.com/Kong/kong/issues/6583) in Kong repository with all the info. They pointed out that you may not be using their [openresty-build-tools](https://github.com/Kong/kong-build-tools/tree/master/openresty-build-tools) for building Kong and that's maybe the cause of the error.

**Additional environment details (AWS, VirtualBox, Docker for MAC, physical, etc.):**

I'm running this as a helm chart on kubernetes.

**Comment by miguelaeh** [comment]
Hi @alesanmed ,
Thank you very much for reporting this. I will open an internal task to check it in deeper and fix whatever is needed at build time.
Regards.

**Comment by FraPazGal** [comment]
Hello @alesanmed,

We have just released a new version of this container, `2.2.1-debian-10-r1`, which has OpenResty updated to the recommended version `1.17` and all the required patches for kong  applied. Could you please try it and check whether the issue is solved on your end?

**Comment by alesanmed** [comment]
@FraPazGal The error is gone! Thanks a million! Closing the issue.
